operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -345,16 +345,16 @@ metadata:
     capabilities: Deep Insights
     categories: Streaming & Messaging
     certified: 'false'
-    containerImage: registry.redhat.io/amq7/amq-streams-rhel7-operator@sha256:3afc6286c8635a6ebc09daeb9f5920a2f2614dafd676c9feafd07e1f90bbec75
+    containerImage: registry.redhat.io/amq7/amq-streams-rhel7-operator@sha256:e6760e7ba4db2facff687af0c7113b22ecccb1af772f1174da1611deb2deb50b
     createdAt: 2021-03-23 13:13:13
     description: Red Hat AMQ Streams is a massively scalable, distributed, and high performance data streaming platform based on Apache Kafka®.
     repository: https://github.com/strimzi/strimzi-kafka-operator
     support: Red Hat
     operators.openshift.io/infrastructure-features: |-
-      ["Disconnected", "Proxy"]
+      ["disconnected", "proxy"]
```

Comment:
I think you should keep it as it is. It works now and from Slack discussions it is not clear what is wrong.